i18n_alchemy.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
   s.test_files    = Dir["test/**/*"]
   s.require_paths = ["lib"]
 
-  s.add_dependency "activesupport", ">= 3.2.0", "<= 4.2.0"
+  s.add_dependency "activesupport", ">= 3.2.0", "<= 4.2.1"
```

Comment:
Yeah, we should work with anything in the 4.x series, so `< 5` should work I think :+1: